recipes/spdlog/all/conanfile.py

```diff
@@ -33,23 +34,26 @@ class SpdlogConan(ConanFile):
         "wchar_support": False,
         "wchar_filenames": False,
         "no_exceptions": False,
+        "fmt_header_only": False,
     }
+    generators = "CMakeDeps"
```

Comment:
not really like this change, I prefer explicit declaration of generators in one place